### PR TITLE
ISSUE-69: Upload private corpus to the database

### PR DESCRIPTION
**Pull request by alimuhammadahmer**
This is a pull request for uploading a private corpus (which is a list of sentences) to database.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,17 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/regex.ym
...
 ```
+
+### Upload private corpus to the database
+
+https://github.com/buerokratt/Data-Anonymizer/issues/69
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/insert_corpora_task.sql
+
+```
+```
```
Comment (turnerrainer) [inline README.md:24]: Please add a sample curl request.

File: README.md [hunk 2]
```diff
@@ -13,3 +13,17 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/regex.ym
...
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/corpora.yml
+
+```
+```
```
Comment (turnerrainer) [inline README.md:29]: Please add a sample curl request.

**Review by turnerrainer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alimuhammadahmer) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (alimuhammadahmer) [reply README.md]: Done

File: DSL.Resql/insert_corpora_task.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+insert into Corpora_Tasks (raw_text, corpora_id, is_private, created_at) VALUES (:raw_text, :corpora_id, :is_private, :created_at::timestamp) RETURNING id;
```
Comment (turnerrainer) [inline DSL.Resql/insert_corpora_task.sql:1]: No newline at end of file

**Review by turnerrainer — CHANGES_REQUESTED** [review]
No newline at end of file, otherwise OK

File: DSL.Resql/insert_corpora_task.sql (hunk 3 above)
Comment (alimuhammadahmer) [reply DSL.Resql/insert_corpora_task.sql:1]: Done

**Review by turnerrainer — APPROVED** [review]

**Pull request merged**